modules/tinymce/changelog.txt

```diff
@@ -30,6 +30,7 @@ Version 5.4.0 (TBD)
     Fixed the `autolink` feature so that it no longer treats a string with multiple "@" characters as an email address #TINY-4773
     Fixed an issue where removing the editor would leave unexpected attributes on the target element #TINY-4001
     Fixed the `link` plugin now suggest `mailto:` when the text contains an '@' without URL separators (/'s) #TINY-5941
+    Fixed valid custom element names can now contain a wider range of characters #5971
```

Comment:
This is missing `TINY-` in the number and this comment is still far too broad. TinyMCE has always allowed the full range for custom elements, it's only the single `valid_children` setting it didn't work for. So this should just refer to that.

Reply:
I was approaching it from the angle that if not every codepath/use supported it then it's not supported (or is supported with caveats), but I totally see what you mean. Changing and adding the TINY-.

Reply:
Looks good, thanks! :+1: